app/assets/javascripts/orders.coffee

```diff
@@ -40,15 +40,18 @@ findLastCategory = ->
   orders = $('.well').find '.order'
   $(orders[orders.length-1]).find '#category'
 
-$(document).on "change", ".form-control", (e) ->
+$(document).on "change", ".orders-list .form-control", (e) ->
```

Comment:
Ooops, the element this references was actually removed.  I'll remove this event.